clients/vscode-hlasmplugin/.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 lib
 hlasmplugin.log
-bin
```

Comment:
Why is bin removed? The contents of bin (language server executables) should never be checked into the git

Reply:
This change for the build/bin folder, not bin/. Im not really sure why but it works, the build/bin files were updated and bin/ was not uploaded.